swagger_to_sdk_config.json

```diff
@@ -18,7 +18,7 @@
       ]
     },
     "azure_mgmt_authorization": {
-      "swagger": "arm-authorization/2015-07-01/swagger/authorization.json",
+      "swagger": "specification/authorization/resource-manager/readme.md",
```

Comment:
Python has it wrong. I have already tagged @lmazuel and will work with him regarding python and approval specifically for this file.